composer.json

```diff
@@ -20,15 +20,20 @@
         "source": "https://github.com/there4/slim-test-helpers"
     },
     "require": {
-        "slim/slim": "2.6.*",
-        "phpunit/phpunit": "^4.8|5.*",
-        "phpunit/dbunit": "1.*|2.*",
-        "illuminate/database": ">=4.0"
+        "php": ">=8.2.0",
+        "slim/slim": "~2.8.0",
```

Comment:
Have you tested upgrading this to ^3.13.1?  I'm running into a conflict with Leadgen where this specifies 2.8.0 and asnet-api specifies 3.13.1